### ISSUE TITLE
BoxLegalHoldPolicy.create(BoxAPIConnection api, String name) is omitting the is_ongoing boolean

### ISSUE THREAD

**Issue by kylefernandadams** [box/box-java-sdk#652]
- [x] I have checked that the [SDK documentation][sdk-docs] and [API documentation][api-docs] doesn't solve my issue

### Description of the Issue
The Create Legal Hold Policy endpoint requires one of the two options:
1)  filter_started_at date/time AND filter_ended_at date/time 
2) is_ongoing boolean

### API Documentation
[Create Legal Hold Policy](https://developer.box.com/v2.0/reference#create-new-legal-hold-policy)

> A date parameter is required: either a date range ("filter_started_at" and "filter_ended_at") or "is_ongoing". The date range should be in the past ("filter_ended_at" should be today's date or before today's date).

### Methods
[BoxLegalHoldPolicy.create(BoxAPIConnection api, String name, String description, Date filterStartedAt, Date filterEndedAt)](https://github.com/box/box-java-sdk/blob/master/src/main/java/com/box/sdk/BoxLegalHoldPolicy.java#L87):  works as expected.

[BoxLegalHoldPolicy.create(BoxAPIConnection api, String name)](https://github.com/box/box-java-sdk/blob/master/src/main/java/com/box/sdk/BoxLegalHoldPolicy.java#L74): does not add the is_ongoing boolean.

### Versions Used
Java SDK: 2.23.2
Java: 9.0.1

### Steps to Reproduce
Run the following code snippet:
```
BoxLegalHoldPolicy.Info boxLegalHoldPolicyInfo = BoxLegalHoldPolicy.create(api, "Example Legal Hold");
```

### Error Message, Including Stack Trace
```
com.box.sdk.BoxAPIResponseException: The API returned an error code [400 | ekdjcnfws5cwdww0] bad_request - Bad Request
        at com.box.sdk.BoxAPIResponse.<init>(BoxAPIResponse.java:92)
        at com.box.sdk.BoxJSONResponse.<init>(BoxJSONResponse.java:32)
        at com.box.sdk.BoxAPIRequest.trySend(BoxAPIRequest.java:558)
        at com.box.sdk.BoxAPIRequest.send(BoxAPIRequest.java:333)
        at com.box.sdk.BoxAPIRequest.send(BoxAPIRequest.java:308)
        at com.box.sdk.BoxLegalHoldPolicy.create(BoxLegalHoldPolicy.java:103)
        at com.box.sdk.BoxLegalHoldPolicy.create(BoxLegalHoldPolicy.java:75)
        at com.box.developer.training.exercise8.BoxLegalHolds.createLegalHold(BoxLegalHolds.java:51)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:209)
        at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:136)
        at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:102)
        at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:891)
        at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:797)
        at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87)
        at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:991)
        at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:925)
        at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:974)
        at org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:866)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:635)
        at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:851)
        at javax.servlet.http.HttpServlet.service(HttpServlet.java:742)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:99)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.springframework.web.filter.HttpPutFormContentFilter.doFilterInternal(HttpPutFormContentFilter.java:109)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.springframework.web.filter.HiddenHttpMethodFilter.doFilterInternal(HiddenHttpMethodFilter.java:93)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:200)
        at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
        at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
        at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:198)
        at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
        at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493)
        at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:140)
        at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
        at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
        at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:342)
        at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:800)
        at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
        at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:806)
        at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1498)
        at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
        at java.base/java.lang.Thread.run(Thread.java:834)
```

[sdk-docs]: ./doc
[api-docs]: https://developer.box.com/docs

**Comment by carycheng** [comment]
Thanks for letting us know Kyle! We'll take a look at this the next couple of days

**Comment by carycheng** [comment]
Hey @kylefernandadams is issue has been resolved in #653, let us know if this works. Thank you!

**Comment by kylefernandadams** [comment]
@carycheng https://github.com/box/box-java-sdk/blob/master/src/main/java/com/box/sdk/BoxLegalHoldPolicy.java#L75 needs to call the new createOngoing method or else it will throw a 400 exception.
